src/material-experimental/mdc-helpers/_focus-indicators-theme.scss

```diff
@@ -27,7 +27,7 @@
   }
   @else {
     $theme: theming.private-legacy-get-theme($theme-or-color-config-or-color);
-    @include theming.private-check-duplicate-theme-styles($theme, 'mat-mdc-focus-indicators') {
+    @include theming.private-check-duplicate-theme-styles($theme, 'mat-focus-indicators') {
```

Comment:
I think that this one should keep the `mdc` since the string is used to verify that styles aren't duplicated.

Reply:
Done, thanks for the suggestion!